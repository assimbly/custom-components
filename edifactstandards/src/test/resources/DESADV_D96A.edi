UNA:+.? 'UNB+UNOC:3+3779000001320:14+8717703610000:14+170302:1456+14748890++M51'UNH+1+DESADV:D:96A:UN:EAN005'BGM+351::9+1700376600000050225+9'DTM+137:201703021456:203'DTM+11:201703021555:203'DTM+2:201703020000:203'DTM+17:201703020000:203'RFF+ON:123456'RFF+HN:123456'NAD+BY+3025570006402::9'NAD+SU+8717703610000::9'NAD+DP+3025570006709::9'NAD+SF+3030001811814::9'NAD+WH+3030001811814::9'CPS+1'PAC+1++X1::9'CPS+2+1'PAC+1++X1::9'PCI+33E'GIN+BJ+302557011015644620'CPS+3+2'PAC+1++CT'PCI+36E'GIN+BX+2016411545'PCI+38E'DTM+36:20180828:102'LIN+2++8934746053896:EN+++00'PIA+1+5076:VN'PIA+1+5076:SA'QTY+12:1'QTY+59:12'PCI+33E'GIN+BJ+187177036170072591'CPS+4+2'PAC+1++CT'PCI+36E'GIN+BX+2016484400'PCI+38E'DTM+36:20180327:102'LIN+1++28850367100020:EN+++00'PIA+1+8074:VN'PIA+1+8074:SA'QTY+12:5'QTY+59:12'PCI+33E'GIN+BJ+187177036170070986'CPS+5+2'PAC+7++CT'PCI+36E'GIN+BX+2016484400'PCI+38E'DTM+36:20180327:102'LIN+1++28850367100020:EN+++00'PIA+1+8074:VN'PIA+1+8074:SA'QTY+12:84'QTY+59:12'PCI+33E'GIN+BJ+187177036170070986'UNT+59+1'UNZ+1+14748890'