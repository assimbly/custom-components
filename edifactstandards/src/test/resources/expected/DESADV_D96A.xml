<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<Envelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.blujaysolutions.com/2018/DESADV file:///C:/Users/jss/Documents/seacon/germany_interfaces/develop/desadv_D96A/DESADV_D96A.xsd" xmlns="http://www.blujaysolutions.com/2018/DESADV">
    <Interchange>
        <UNB>
            <F0020>14748890</F0020>
            <F0026>M51</F0026>
            <S001>
                <F0001>UNOC</F0001>
                <F0002>3</F0002>
            </S001>
            <S002>
                <F0004>3779000001320</F0004>
                <F0007>14</F0007>
            </S002>
            <S003>
                <F0010>8717703610000</F0010>
                <F0007>14</F0007>
            </S003>
            <S004>
                <F0017>170302</F0017>
                <F0019>1456</F0019>
            </S004>
        </UNB>
        <UNZ>
            <F0036>1</F0036>
            <F0020>14748890</F0020>
        </UNZ>
        <Group>
            <Message_DESADV>
                <UNH>
                    <F0062>1</F0062>
                    <S009>
                        <F0065>DESADV</F0065>
                        <F0052>D</F0052>
                        <F0054>96A</F0054>
                        <F0051>UN</F0051>
                        <F0057>EAN005</F0057>
                    </S009>
                </UNH>
                <BGM>
                    <F1004>1700376600000050225</F1004>
                    <F1225>9</F1225>
                    <C002>
                        <F1001>351</F1001>
                        <F3055>9</F3055>
                    </C002>
                </BGM>
                <DTM>
                    <C507>
                        <F2005>137</F2005>
                        <F2380>201703021456</F2380>
                        <F2379>203</F2379>
                    </C507>
                </DTM>
                <DTM>
                    <C507>
                        <F2005>11</F2005>
                        <F2380>201703021555</F2380>
                        <F2379>203</F2379>
                    </C507>
                </DTM>
                <DTM>
                    <C507>
                        <F2005>2</F2005>
                        <F2380>201703020000</F2380>
                        <F2379>203</F2379>
                    </C507>
                </DTM>
                <DTM>
                    <C507>
                        <F2005>17</F2005>
                        <F2380>201703020000</F2380>
                        <F2379>203</F2379>
                    </C507>
                </DTM>
                <UNT>
                    <F0074>59</F0074>
                    <F0062>1</F0062>
                </UNT>
                <SG1>
                    <RFF>
                        <C506>
                            <F1153>ON</F1153>
                            <F1154>123456</F1154>
                        </C506>
                    </RFF>
                </SG1>
                <SG1>
                    <RFF>
                        <C506>
                            <F1153>HN</F1153>
                            <F1154>123456</F1154>
                        </C506>
                    </RFF>
                </SG1>
                <SG2>
                    <NAD>
                        <F3035>BY</F3035>
                        <C082>
                            <F3039>3025570006402</F3039>
                            <F3055>9</F3055>
                        </C082>
                    </NAD>
                </SG2>
                <SG2>
                    <NAD>
                        <F3035>SU</F3035>
                        <C082>
                            <F3039>8717703610000</F3039>
                            <F3055>9</F3055>
                        </C082>
                    </NAD>
                </SG2>
                <SG2>
                    <NAD>
                        <F3035>DP</F3035>
                        <C082>
                            <F3039>3025570006709</F3039>
                            <F3055>9</F3055>
                        </C082>
                    </NAD>
                </SG2>
                <SG2>
                    <NAD>
                        <F3035>SF</F3035>
                        <C082>
                            <F3039>3030001811814</F3039>
                            <F3055>9</F3055>
                        </C082>
                    </NAD>
                </SG2>
                <SG2>
                    <NAD>
                        <F3035>WH</F3035>
                        <C082>
                            <F3039>3030001811814</F3039>
                            <F3055>9</F3055>
                        </C082>
                    </NAD>
                </SG2>
                <SG10>
                    <CPS>
                        <F7164>1</F7164>
                    </CPS>
                    <SG11>
                        <PAC>
                            <F7224>1</F7224>
                            <C202>
                                <F7065>X1</F7065>
                                <F3055>9</F3055>
                            </C202>
                        </PAC>
                    </SG11>
                </SG10>
                <SG10>
                    <CPS>
                        <F7164>2</F7164>
                        <F7166>1</F7166>
                    </CPS>
                    <SG11>
                        <PAC>
                            <F7224>1</F7224>
                            <C202>
                                <F7065>X1</F7065>
                                <F3055>9</F3055>
                            </C202>
                        </PAC>
                        <SG13>
                            <PCI>
                                <F4233>33E</F4233>
                            </PCI>
                            <SG14>
                                <GIN>
                                    <F7405>BJ</F7405>
                                    <C208>
                                        <F7402>302557011015644620</F7402>
                                    </C208>
                                </GIN>
                            </SG14>
                        </SG13>
                    </SG11>
                </SG10>
                <SG10>
                    <CPS>
                        <F7164>3</F7164>
                        <F7166>2</F7166>
                    </CPS>
                    <SG11>
                        <PAC>
                            <F7224>1</F7224>
                            <C202>
                                <F7065>CT</F7065>
                            </C202>
                        </PAC>
                        <SG13>
                            <PCI>
                                <F4233>36E</F4233>
                            </PCI>
                            <SG14>
                                <GIN>
                                    <F7405>BX</F7405>
                                    <C208>
                                        <F7402>2016411545</F7402>
                                    </C208>
                                </GIN>
                            </SG14>
                        </SG13>
                        <SG13>
                            <PCI>
                                <F4233>38E</F4233>
                            </PCI>
                            <DTM>
                                <C507>
                                    <F2005>36</F2005>
                                    <F2380>20180828</F2380>
                                    <F2379>102</F2379>
                                </C507>
                            </DTM>
                        </SG13>
                    </SG11>
                    <SG15>
                        <LIN>
                            <F1082>2</F1082>
                            <F7083>00</F7083>
                            <C212>
                                <F7140>8934746053896</F7140>
                                <F7143>EN</F7143>
                            </C212>
                        </LIN>
                        <PIA>
                            <F4347>1</F4347>
                            <C212>
                                <F7140>5076</F7140>
                                <F7143>VN</F7143>
                            </C212>
                        </PIA>
                        <PIA>
                            <F4347>1</F4347>
                            <C212>
                                <F7140>5076</F7140>
                                <F7143>SA</F7143>
                            </C212>
                        </PIA>
                        <QTY>
                            <C186>
                                <F6063>12</F6063>
                                <F6060>1</F6060>
                            </C186>
                        </QTY>
                        <QTY>
                            <C186>
                                <F6063>59</F6063>
                                <F6060>12</F6060>
                            </C186>
                        </QTY>
                        <SG20>
                            <PCI>
                                <F4233>33E</F4233>
                            </PCI>
                            <SG21>
                                <GIN>
                                    <F7405>BJ</F7405>
                                    <C208>
                                        <F7402>187177036170072591</F7402>
                                    </C208>
                                </GIN>
                            </SG21>
                        </SG20>
                    </SG15>
                </SG10>
                <SG10>
                    <CPS>
                        <F7164>4</F7164>
                        <F7166>2</F7166>
                    </CPS>
                    <SG11>
                        <PAC>
                            <F7224>1</F7224>
                            <C202>
                                <F7065>CT</F7065>
                            </C202>
                        </PAC>
                        <SG13>
                            <PCI>
                                <F4233>36E</F4233>
                            </PCI>
                            <SG14>
                                <GIN>
                                    <F7405>BX</F7405>
                                    <C208>
                                        <F7402>2016484400</F7402>
                                    </C208>
                                </GIN>
                            </SG14>
                        </SG13>
                        <SG13>
                            <PCI>
                                <F4233>38E</F4233>
                            </PCI>
                            <DTM>
                                <C507>
                                    <F2005>36</F2005>
                                    <F2380>20180327</F2380>
                                    <F2379>102</F2379>
                                </C507>
                            </DTM>
                        </SG13>
                    </SG11>
                    <SG15>
                        <LIN>
                            <F1082>1</F1082>
                            <F7083>00</F7083>
                            <C212>
                                <F7140>28850367100020</F7140>
                                <F7143>EN</F7143>
                            </C212>
                        </LIN>
                        <PIA>
                            <F4347>1</F4347>
                            <C212>
                                <F7140>8074</F7140>
                                <F7143>VN</F7143>
                            </C212>
                        </PIA>
                        <PIA>
                            <F4347>1</F4347>
                            <C212>
                                <F7140>8074</F7140>
                                <F7143>SA</F7143>
                            </C212>
                        </PIA>
                        <QTY>
                            <C186>
                                <F6063>12</F6063>
                                <F6060>5</F6060>
                            </C186>
                        </QTY>
                        <QTY>
                            <C186>
                                <F6063>59</F6063>
                                <F6060>12</F6060>
                            </C186>
                        </QTY>
                        <SG20>
                            <PCI>
                                <F4233>33E</F4233>
                            </PCI>
                            <SG21>
                                <GIN>
                                    <F7405>BJ</F7405>
                                    <C208>
                                        <F7402>187177036170070986</F7402>
                                    </C208>
                                </GIN>
                            </SG21>
                        </SG20>
                    </SG15>
                </SG10>
                <SG10>
                    <CPS>
                        <F7164>5</F7164>
                        <F7166>2</F7166>
                    </CPS>
                    <SG11>
                        <PAC>
                            <F7224>7</F7224>
                            <C202>
                                <F7065>CT</F7065>
                            </C202>
                        </PAC>
                        <SG13>
                            <PCI>
                                <F4233>36E</F4233>
                            </PCI>
                            <SG14>
                                <GIN>
                                    <F7405>BX</F7405>
                                    <C208>
                                        <F7402>2016484400</F7402>
                                    </C208>
                                </GIN>
                            </SG14>
                        </SG13>
                        <SG13>
                            <PCI>
                                <F4233>38E</F4233>
                            </PCI>
                            <DTM>
                                <C507>
                                    <F2005>36</F2005>
                                    <F2380>20180327</F2380>
                                    <F2379>102</F2379>
                                </C507>
                            </DTM>
                        </SG13>
                    </SG11>
                    <SG15>
                        <LIN>
                            <F1082>1</F1082>
                            <F7083>00</F7083>
                            <C212>
                                <F7140>28850367100020</F7140>
                                <F7143>EN</F7143>
                            </C212>
                        </LIN>
                        <PIA>
                            <F4347>1</F4347>
                            <C212>
                                <F7140>8074</F7140>
                                <F7143>VN</F7143>
                            </C212>
                        </PIA>
                        <PIA>
                            <F4347>1</F4347>
                            <C212>
                                <F7140>8074</F7140>
                                <F7143>SA</F7143>
                            </C212>
                        </PIA>
                        <QTY>
                            <C186>
                                <F6063>12</F6063>
                                <F6060>84</F6060>
                            </C186>
                        </QTY>
                        <QTY>
                            <C186>
                                <F6063>59</F6063>
                                <F6060>12</F6060>
                            </C186>
                        </QTY>
                        <SG20>
                            <PCI>
                                <F4233>33E</F4233>
                            </PCI>
                            <SG21>
                                <GIN>
                                    <F7405>BJ</F7405>
                                    <C208>
                                        <F7402>187177036170070986</F7402>
                                    </C208>
                                </GIN>
                            </SG21>
                        </SG20>
                    </SG15>
                </SG10>
            </Message_DESADV>
        </Group>
    </Interchange>
</Envelope>